### DISCUSSION TITLE
Installation help Shared IP

### DISCUSSION THREAD

**Discussion by McflySamuel** [eusonlito/GPS-Tracker#47] [Q&A]
I am using a Shared IP Address for my websites, and I would like to know if this application would work. Could it be compatible?

**Comment by eusonlito** [comment]
Yes, the only thing you need to do is to generate servers on ports that you have free on the server.

**Reply by McflySamuel** [reply]
i have another question do i need configure the docker-compose.yml file

**Reply by eusonlito** [reply]
Yes, you can 😅

**Comment by McflySamuel** [comment]
Yes i know i can. But will this still work? if i don't configure.

**Reply by eusonlito** [reply]
It will work with and without configuration, is an optional step:

```
[OPTIONAL] Copy file docker/docker-compose.yml.example to docker/docker-compose.yml and configure your own settings
```
